packages/@ember/-internals/glimmer/lib/resolver.ts

```diff
@@ -422,7 +366,7 @@ export default class RuntimeResolverImpl implements RuntimeResolver<OwnedTemplat
       return null;
     }
 
-    return (args, vm) => {
+    return (args: { capture: () => any }, vm: { associateDestroyable: (arg0: {}) => void }) => {
```

Comment:
I think these types can be imported from `@glimmer/runtime` (instead of having to type them manually in the signature), I'm curious why this needed to be changed (did you have warnings or something?)...

Reply:
after vscode restart I did not get such errors, removing autocased definitions